vipps-ecom-api-faq.md

```diff
@@ -884,6 +884,8 @@ for problems with the callbacks. The API Dashboard is under "Utvikler".
 If you need help solving a callback-related problem, please send us a
 complete HTTP request, and any other related details, so we can investigate.
 
+**Please note:** Cllback URLs _must_ use HTTPS.
```

Comment:
```suggestion
**Please note:** Callback URLs _must_ use HTTPS.
```

Reply:
Alrteady done! 🤦‍♂️

Reply:
emmm no its still here, you must have undone it

Reply:
Ugh, missed the FAQ file.